tests/degradation_test_v2.py

```diff
@@ -0,0 +1,175 @@
+""" Degradation Module Tests. """
```

Comment:
Since this is the file we are going with, you can remove the other two files and rename this to tests/degradation_test.py.